extensions/mssql/package.json

```diff
@@ -551,6 +570,16 @@
           "when": "connectionProvider == MSSQL && nodeType =~ /^(ServerLevelLogin|User)$/ && config.workbench.enablePreviewFeatures",
           "group": "connection@2"
         },
+        {
+          "command": "mssql.renameObject",
+          "when": "connectionProvider == MSSQL && nodeType =~ /^(Database|ServerLevelLogin|User|Table|View)$/ && config.workbench.enablePreviewFeatures",
+          "group": "connection@3"
+        },
+        {
+          "command": "mssql.renameObject",
```

Comment:
Column is also available in view which doesn't support renaming, so only enable it for table columns.